package.json

```
@@ -10,7 +10,9 @@
   },
   "scripts": {
     "heroku-postbuild": "gulp build:prod",
```

Maybe update this to call the new build script too:

```suggestion
    "heroku-postbuild": "npm run build:prod",
```